cmd/juju/machine/upgradeseries.go

```diff
@@ -212,7 +206,7 @@ func (c *upgradeSeriesCommand) Run(ctx *cmd.Context) error {
 // dependency this function should contain minimal logic other than gathering an
 // API handle and making the API call.
 func (c *upgradeSeriesCommand) UpgradeSeriesPrepare(ctx *cmd.Context) (err error) {
-	apiRoot, err := c.ensureAPIClients()
+	apiRoot, err := c.ensureAPIClient()
```

Comment:
Is it worth designing into this experience that it prints out what applications will be pinned as a result of putting this machine into prepared state?
I realize it isn't driven by the client, but is that useful feedback to give to the operator?

Reply:
This is simple to do without going back to the server. It is similar to how we generated the list of apps when pinning in the client prior.